bin/detail/toolchain_table.py

```diff
@@ -576,6 +576,10 @@ def verify(self):
       Toolchain('osx-10-12-hid-sections', 'Xcode', osx_version='10.12'),
       Toolchain('osx-10-12-lto', 'Xcode', osx_version='10.12'),
       Toolchain('osx-10-12-cxx98', 'Xcode', osx_version='10.12'),
+      Toolchain('osx-10-12-cxx14', 'Xcode', osx_version='10.12'),
+      Toolchain('osx-10-12-cxx14-fpic', 'Xcode', osx_version='10.12'),
```

Comment:
I think all `osx-*` toolchains by default have PIC. Have you seen any difference between `osx-10-12-cxx14` and `osx-10-12-cxx14-fpic`? Like fixing some linker errors.

Reply:
I’m honestly not sure. I never had any linked issues or anything on OS X specifically - I just added these prematurely. I can definitely remove to avoid the clutter